src/datacatalog/plugins/dcat_ap_ams/dataset.py

```diff
@@ -189,4 +189,13 @@
         title="Sorteerdatum",
         read_only=True
     )
+).add(
```

Comment:
Deze `ams:status` betreft, als ik het goed begrijp, niet de status van de dataset zelf, maar de status van het "kaartje" in onze datacatalogus.

In dat geval hoort `ams:status` niet "top-level" te staan, zoals hier, maar ondergebracht te worden in `foaf:isPrimaryTopicOf`, naast `issued` en `modified`.

Reply:
Ik weet niet of het onderscheid tussen de dataset en het kaartje van de dataset goed zie. Het is voornamelijk een middel om datasets onzichtbaar te kunnen maken in productie, bijv als de link niet klopt . Zie ook adms:status  in 

https://dcat-ap-donl.readthedocs.io/en/latest/classes-dataset.html

Reply:
Dat gaat volgens mij dus inderdaad er de status van de dataset zelf, niet over de catalogus-entry. Zou je dit toch nog willen aanpassen?